teitelbot/response/action.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum
+
+
+class Action(Enum):
+    NONE = 0
+    EMAIL_ACTION = 1
+    HAVING_ACTION = 2
+    REPEAT_SARCASTICALLY_ACTION = 3
+    DEFAULT_ACTION = 5
```

Comment:
Is there supposed to be 4?

Reply:
my bad.

Reply:
what about:
`Action = Enum(
    "NONE",
    "EMAIL_ACTION",
...
)`

Reply:
That might be easier to maintain, but choose whichever is more conventional

Reply:
sorry couldn't make it format code

Reply:
Meh

Reply:
Is = auto() an option?

Reply:
what do u mean?

Reply:
https://stackoverflow.com/questions/58471615/is-it-possible-to-define-enums-without-values-with-python

Reply:
nice. will do

Reply:
If it's not too ugly